### PR TITLE
(DOCSP-754): Added default settings.

### PR DESCRIPTION
**Pull request by atsansone**
@i80and : My reading of the ticket puts this as a reference change. Please have a look.

### CODE REVIEW

File: source/includes/options-mongosqld.yaml [hunk 1]
```diff
@@ -176,6 +176,36 @@ description: |
 optional: true
 ---
 program: mongosqld
+name: defaultAuthDatabase
+directive: option
+args: <database>
+description: |
+
+  Specifies the default source database for |bi| users. Set this value
```
Comment (i80and) [inline source/includes/options-mongosqld.yaml]: This phrasing is a little coy. I might replace the first two sentences with something like

```
Specifies the default authentication source database for connecting to the |bi|.
```

Comment (atsansone) [inline source/includes/options-mongosqld.yaml]: Fixed.

**Comment by atsansone** [comment]
@i80and : Back to you for OK.

**Comment by atsansone** [comment]
@jyemin : Could you have a look? The default settings were added.

**Pull request merged**